dfaligner/dataset.py

```diff
@@ -135,7 +140,23 @@ def __getitem__(self, index):
             .squeeze()
             .transpose(0, 1)
         )  # [mel_bins, frames] -> [frames, mel_bins]
-        text_tokens = self._load_file(basename, speaker, language, "text", "text.pt")
+        if (
+            self.config.model.target_text_representation_level
+            == TargetTrainingTextRepresentationLevel.characters
+        ):
+            text_tokens = torch.Tensor(
+                self.text_processor.encode_escaped_string_sequence(
+                    item["character_tokens"]
+                )
+            ).long()
+        elif self.config.model.target_text_representation_level in [
+            TargetTrainingTextRepresentationLevel.ipa_phones,
+            TargetTrainingTextRepresentationLevel.phonological_features,
+        ]:
+            text_tokens = torch.Tensor(
+                self.text_processor.encode_escaped_string_sequence(item["phone_tokens"])
```

Comment:
Add an else statement that throws an error.
This should never happen unless we add a new `TargetTrainingTextRepresentationLevel` in the future where `text_tokens` would be undefined for the rest of the code.
```suggestion
                else:
                    raise NotImplemented(f"{self.config.model.target_text_representation_level} have not yet been implemented.")
                self.text_processor.encode_escaped_string_sequence(item["phone_tokens"])
```